### DISCUSSION TITLE
Input Quantity value mapping in data object importer[Feature]:

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#14590] [General]
### Feature description

The field name is not visible while mapping because the data object importer cannot map the input quantity value.

Does anyone have the answer?

The input is 10 kg, and the data component type is input quantity value.
My Pimcore community version is 10.5.11

**Comment by mattamon** [comment]
Hey @arulselvan008 did you really wanted to create an issue for this, or do you rather want to open a discussion?
We can transfer the issue into the discussions if you want input from the community :)

**Reply by arulselvan008** [reply]
Anyone with a fix for this problem? I couldn't map the input quantity value data type in the data object importer; please share the pipeline procedure.
